plug.vim

```diff
@@ -799,7 +799,7 @@ function! s:bang(cmd, ...)
     let cmd = a:0 ? s:with_cd(a:cmd, a:1) : a:cmd
     if s:is_win
       let batchfile = tempname().'.bat'
-      call writefile(['@echo off', cmd], batchfile)
+      call writefile(["@echo off\r", cmd . "\r"], batchfile)
```

Comment:
Restrict to Windows only with `(has('win32') ? "\r" : '')`.

Reply:
`has('win32')` is unnecessary because `if s:is_win` means it.

Reply:
You're right. I forgot about my changes.